hazelcast/src/main/java/com/hazelcast/map/impl/wan/WanMapRemoveEvent.java

```diff
@@ -105,7 +122,18 @@ public int getClassId() {
     }
 
     @Override
-    public void incrementEventCount(WanEventCounters counters) {
+    public void incrementEventCount(@Nonnull WanEventCounters counters) {
         counters.incrementRemove(mapName);
     }
+
+    @Nonnull
+    @Override
+    public WanEventService getService() {
```

Comment:
why we need this method? As far a i see it is only used in tests.

Reply:
It's because of this part:
> Also, moved the getServiceName method to private API. It would return a
> String service name which would then force the user to check it against
> private API - either MapService#SERVICE_NAME or ICacheService#SERVICE_NAME.
> Instead, we move provide another method which returns a public enum.

Reply:
I guess we could use it internally ourselves. I'll try to change it.

Reply:
Fixed. We now use this method directly in some places but other places use the default interface method that will return the service name. This is so that we don't have to do the mapping from `WanEventService -> serviceName` in multiple classes. Feel free to take a look.